R/get_oaproc.R

```diff
@@ -147,11 +147,8 @@ get_xs_point <- function(point, width, num_pts) {
 
 #' Get Cross Section Endpoints (experimental)
 #' @description Uses a cross section retrieval web services to retrieve a
-#' cross section between two endpoints.
-#' @param point1 sfc POINT including crs as created by:
-#' \code{sf::st_sfc(sf::st_point(.. ,..), crs)}
-#' @param point2 sfc POINT including crs.
-#' @param num_pts numeric number of points to retrieve along the cross section.
```

Comment:
Looks like num_pts got dropped? How were you thinking about handling this?

Reply:
I'll add it back in. Must of deleted in when changing... Thanks.